### PR TITLE
AVX-37128: global vpc excluded instance and tagging settings

### PR DESCRIPTION
**Pull request by shuoliugit**

### CODE REVIEW

File: aviatrix/resource_aviatrix_global_vpc_excluded_instance.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+			},
+			"region": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Region of the instance.",
+			},
+			"uuid": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "UUID of the instance.",
```
Comment (arvindsree) [inline aviatrix/resource_aviatrix_global_vpc_excluded_instance.go]: This is the UUID of exclude list object

File: aviatrix/resource_aviatrix_global_vpc_excluded_instance_test.go [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+	account_name = aviatrix_account.test_acc_gcp.account_name
+	gw_name      = "tfg-gcp-%[1]s"
+	vpc_id       = "%[4]s"
+	vpc_reg      = "%[5]s"
+	gw_size      = "%[6]s"
+	subnet       = "%[7]s"
+}
+resource "aviatrix_global_vpc_excluded_instance" "test" {
+	account_name  = aviatrix_account.test_acc_gcp.account_name
+	instance_name = aviatrix_transit_gateway.test_transit_gateway_gcp.gw_name
```
Comment (arvindsree) [inline aviatrix/resource_aviatrix_global_vpc_excluded_instance_test.go:72]: Ideally this should not be aviatrix instances. but this is ok to just test the exclude list part

File: aviatrix/resource_aviatrix_global_vpc_tagging_settings.go [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+		DeleteWithoutTimeout: resourceAviatrixGlobalVpcTaggingSettingsDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"service_state": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringInSlice([]string{"semi_automatic", "automatic"}, false),
```
Comment (arvindsree) [inline aviatrix/resource_aviatrix_global_vpc_tagging_settings.go]: "disabled" as well

Comment (arvindsree) [reply aviatrix/resource_aviatrix_global_vpc_tagging_settings.go]: also update in docs

**Review by zhiqiangjin — APPROVED** [review]

**Pull request merged**